results/Proprietary/Generic 3button Macropad/README.md

```diff
@@ -0,0 +1,4 @@
+Test setup:
+
+- Host: Raspberry Pi 4 running Raspberry OS version 12
+- Keyboard: Generic 3 button + rotary encoder macropad from Aliexpress
```

Comment:
If there's no way for someone else to identify what product this data is for, then I don't think this data is very useful. Ideally, we should provide a link to the product, though I don't have much confidence that a link to a generic product on Aliexpress would stay working for very long.

Reply:
Good point. I was just including all the data I collected but this one isn't particularly useful. I'll remove it